index.html

```diff
@@ -39,22 +39,276 @@ <h1 class="main-text font-poppins">
           <ul>
             <li>
               <a href="https://twitter.com/_JavierAybar">
-                <img class="social-media-icon" src="images/Twitter-icon.svg" alt="Twitter" />
+                <img
+                  class="social-media-icon"
+                  src="images/Twitter-icon.svg"
+                  alt="Twitter"
+                />
               </a>
             </li>
             <li>
               <a href="https://www.linkedin.com/in/javier-aybar-932376274/">
-                <img class="social-media-icon" src="images/Linkedin icon.svg" alt="Linkedin" />
+                <img
+                  class="social-media-icon"
+                  src="images/Linkedin icon.svg"
+                  alt="Linkedin"
+                />
               </a>
             </li>
             <li>
               <a href="https://github.com/JavierAybar">
-                <img class="social-media-icon" src="images/Github-icon.svg" alt="GitHub" />
+                <img
+                  class="social-media-icon"
+                  src="images/Github-icon.svg"
+                  alt="GitHub"
+                />
               </a>
             </li>
           </ul>
         </section>
       </section>
     </main>
+    <!--Works-->
```

Comment:
Great work done so far 👍🏼  , but the main tag is better used to wrap all main content of a page only header and footer are separated then inside  main you organize your blocs of code in sections like introduction , works , contact ...